lib/task_helpers.rb

```diff
@@ -17,7 +17,11 @@ def self.rebuild_index(alias_name=nil)
       index_name = create_index()
 
       [Comment, CommentThread].each do |model|
-        model.import(index: index_name)
+        model.import(index: index_name, batch_size: 100) do |response|
```

Comment:
I think the idea was to make batch_size configurable again.  Any idea what the default is?